cjsmin/src/chess.ts

```diff
@@ -1272,11 +1292,18 @@ export class Chess {
               this._board[from].unambiguousSymbol,
               {
                 type: this._board[to].type,
-                unambiguousSymbol: this._board[from].unambiguousSymbol,
+                unambiguousSymbol: this._board[to].unambiguousSymbol,
               },
               BITS.CAPTURE
             );
           } else if (to === this._epSquare) {
+            // for the unambiguous symbol we can get the file and casing
+            const file = algebraic(to).charAt(0);
+            let uas = 'p' + file;
+            if (us === BLACK) {
+              uas = uas.toUpperCase();
```

Comment:
black should be lowercase?

Reply:
think you saw but this is an undoMove, so us = BLACK means them white

Reply:
yep understand now thx